cli/helptext.go

```diff
@@ -445,6 +493,23 @@ func subcommandText(width int, cmd *cmds.Command, rootName string, path []string
 	return lines
 }
 
+// Text printed at the beginning of --help,
+// after 'WARNING: ' tag at the start of the command.
+func generateWarningText(cmd *cmds.Command) string {
+	switch cmd.Status {
+	case cmds.Active:
+		return "" // We don't print a warning for a normal active command.
+	case cmds.Deprecated:
+		return "DEPRECATED, command will be removed in the future"
+	case cmds.Experimental:
+		return "EXPERIMENTAL, command may change in future releases"
+	case cmds.Removed:
+		return "REMOVED, command is no longer available"
```

Comment:
IF this text looks ok, then we can merge and ship this.

Reply:
Yep, looks fair.